inspirehep/dojson/common/rules.py

```diff
@@ -0,0 +1,396 @@
+        method = 'submitter'
+    else:
+        method = normalized_c_value
+
+    return {
+        'date': value.get('d'),
+        'email': value.get('b'),
+        'internal_uid': internal_uid,
+        'method': method,
+        'orcid': orcid,
+        'source': source,
+        'submission_number': value.get('e'),
+    }
+
+
+@hep2marc.over('541', '^acquisition_source$')
+@hepnames2marc.over('541', '^acquisition_source$')
+def acquisition_source2marc(self, key, value):
+    orcid = value.get('orcid')
+    source = value.get('source')
+
+    a_value = 'orcid:' + orcid if orcid else source
+
+    method = value.get('method')
+
+    if method == 'batchuploader':
+        c_value = 'batchupload'
+    elif method == 'submitter':
+        c_value = 'submission'
+    else:
+        c_value = method
+
+    return {
+        'a': a_value,
+        'b': value.get('email'),
+        'c': c_value,
+        'd': value.get('date'),
+        'e': value.get('submission_number'),
+    }
+
+
+@conferences.over('public_notes', '^500..')
+@experiments.over('public_notes', '^500..')
+@hep.over('public_notes', '^500..')
+@hepnames.over('public_notes', '^500..')
+@institutions.over('public_notes', '^500..')
+@jobs.over('public_notes', '^500..')
+@journals.over('public_notes', '^500..')
+def public_notes_500(self, key, value):
+    public_notes = self.get('public_notes', [])
+
+    source = force_single_element(value.get('9', ''))
+    for value in force_force_list(value):
+        for public_note in force_force_list(value.get('a')):
+            public_notes.append({
+                'source': source,
+                'value': public_note,
+            })
+
+    return public_notes
+
+
+@hep2marc.over('500', '^public_notes$')
+@hepnames2marc.over('500', '^public_notes$')
+@utils.for_each_value
+def public_notes2marc(self, key, value):
+    return {
+        '9': value.get('source'),
+        'a': value.get('value'),
+    }
+
+
+@conferences.over('_private_notes', '^595..')
+@experiments.over('_private_notes', '^595..')
+@hep.over('_private_notes', '^595..')
+@hepnames.over('_private_notes', '^595..')
+@institutions.over('_private_notes', '^595..')
+@jobs.over('_private_notes', '^595..')
+@journals.over('_private_notes', '^595..')
+@utils.for_each_value
+def _private_notes_595(self, key, value):
+    return {
+        'source': value.get('9'),
+        'value': value.get('a'),
+    }
+
+
+@hep2marc.over('595', '^_private_notes$')
+@hepnames2marc.over('595', '^_private_notes$')
+@utils.for_each_value
+def _private_notes2marc(self, key, value):
+    return {
+        '9': value.get('source'),
+        'a': value.get('value'),
+    }
+
+
+@conferences.over('inspire_categories', '^65017')
+@experiments.over('inspire_categories', '^65017')
+@hep.over('inspire_categories', '^65017')
+@hepnames.over('inspire_categories', '^65017')
+@institutions.over('inspire_categories', '^65017')
+@jobs.over('inspire_categories', '^65017')
+def inspire_categories(self, key, value):
```

Comment:
Should be solved by https://github.com/inspirehep/inspire-next/pull/1918/commits/21f54124f57e5529fd9f9fc5f930bbbe24feaba4.